src/ListActions/ListActions.js

```diff
@@ -58,202 +45,87 @@ var ListActionActivationSupport = {
 	}
 };
 
-
 /**
-* Fires when the [ListActionsDrawer]{@link module:moonstone/ListActions~ListActionsDrawer} has completed any
-* setup and/or preparation work, e.g., when animating or initial setup. No event-specific
-* information is sent with this event.
-*
-* @event module:moonstone/ListActions~ListActionsDrawer#onComplete
-* @type {Object}
-* @public
-*/
-
-/**
-* {@link module:moonstone/ListActions~ListActionsDrawer} is a [control]{@link module:enyo/Control~Control} used by
+* {@link module:moonstone/ListActions~ListActionsPopup} is a
+* [control]{@link module:moonstone/ContextualPopup~ContextualPopup} used by
 * {@link module:moonstone/ListActions~ListActions} to house a menu of selectable options.
 *
-* @class ListActionsDrawer
-* @extends module:enyo/Control~Control
+* @class ListActionsPopup
+* @extends module:moonstone/ContextualPopup~ContextualPopup
 * @ui
 * @public
```

Comment:
This might have been improperly documented before, but I'm thinking this should be `@private`. cc @webOS101